### PR TITLE
Fix Activity Panel on existing WooCommerce pages

### LINKED ISSUE #726: Activity Panel: Inbox won't open on non Dashboard, non Analytics routes
**Describe the bug**

Clicking on the Activity Panel Inbox icon on pages other than the dashboard ( wp-admin/admin.php?page=wc-admin#/ ) or one of the analytics pages results in console errors and not much else.

```
Uncaught TypeError: Cannot read property 'getNotes' of undefined
    at inbox.js:92
```

**To Reproduce**

1. Go to a page like wp-admin > Woocommerce > Extensions > Subscriptions
2. Click on the Inbox

**Expected behavior**
The inbox should open and display its contents.

Props @belcherj for finding this.

### LINKED ISSUE #738: Unable to add tax rates when WC-Admin is active
**Describe the bug**
When WC-Admin is active, a user is unable to add tax rates to their store

**To Reproduce**

1. Ensure WC-Admin is active
2. Navigate to Settings>Taxes>Standard Tax rates 
3. Click the "insert row"
4. Note nothing happens
5. Same behavior is encountered in all other tax types ie: reduced rates, zero rates, and custom rates

**Expected behavior**
Clicking insert row should add a new row to the table per this screen shot:

screen shot 2018-10-30 at 12 42 15 pm

After disabling WC-Admin, I was able to add tax rates as expected.

### PR DESCRIPTION
**Pull request by emdashcodes**
Fixes #726.
Fixes #738.

This PR fixes a few issues with embedding activity panel on existing WooCommerce pages.

First, it fixes the `Uncaught TypeError` errors that occurs when trying to open notes or orders. This is because the API scripts were not being loaded in the embedded app entry point. It also moves notes over to using Fresh Data to match the orders panel.

Next, it fixes an issue where underscore and lodash where clashing which broke the WordPress.com notifications toolbar. There is a webpack change. Funnily enough, WCS made a similar change that I found through Googleing :). https://github.com/Automattic/woocommerce-services/pull/1522.

Then a few minor CSS issues are fixed. For example, the .com notifications panel getting covered by wc-admin when open.

screen shot 2018-12-05 at 2 03 20 pm

### Detailed test instructions:

* If you don't have any wc-admin notifications in your Inbox, you can force one to show by opening `wc-admin.php` and changing `add_action( 'woocommerce_updated', 'wc_admin_woocommerce_updated' );` to `add_action( 'init', 'wc_admin_woocommerce_updated' );`. Refresh, and then change the line back.
* Test opening various activity panel tabs on any existing WooCommerce page. Also test opening the WordPress.com notifications toolbar.
* Repeat on a dashboard or analytics page.

### CODE REVIEW

File: client/header/style.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	position: fixed;
 	width: 100%;
 	top: 32px;
-	z-index: 99999; /* component-popover is 99990 */
+	z-index: 99991; /* higer than component-popover (99990), lower than notification dropdown at 99999 */
```
Comment (jeffstieler) [inline client/header/style.scss]: Typo in comment.

File: lib/client-assets.php [hunk 2]
```diff
@@ -96,11 +96,13 @@ function wc_admin_register_script() {
...
 	// Resets lodash to wp-admin's version of lodash.
-	wp_add_inline_script(
-		WC_ADMIN_APP,
-		'_.noConflict();',
-		'after'
-	);
+	if ( 'embedded' === $entry ) {
+		wp_add_inline_script(
+			WC_ADMIN_APP,
+			'_ = _.noConflict();',
```
Comment (jeffstieler) [inline lib/client-assets.php]: I'm seeing `Uncaught TypeError: Cannot read property 'noConflict' of undefined` on non-dashboard/analytics pages.

**Comment by jeffstieler** [comment]
This tests well, but I'm seeing an error in console on non-dashboard/analytics pages.

**Review by joshuatf — APPROVED** [review]
Testing well for me and not seeing the error that Jeff mentioned, also running WP 4.9.8 and Gutenberg 4.6.1.

@jeffstieler Could you share which specific pages you were hitting these on?

**Review by timmyc — APPROVED** [review]
This is working well for me, and it fixes #738 also! I updated the original PR description so this will close out that issue on merge too.

**Comment by justinshreve** [comment]
I'm going to get this merged since it makes things better for already broken behavior -- but @jeffstieler if you are still having issues -- can you also provide me with your WooCommerce version and which pages you were testing on? We can follow-up in another PR once I can reproduce.

**Pull request merged**